analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/LiveVariableAnalysisBase.cs

```diff
@@ -77,6 +80,10 @@ protected void Analyze()
             }
             while (queue.Any())
             {
+                if (CancellationToken.IsCancellationRequested)
+                {
+                    return;
+                }
```

Comment:
This is not enough, we need to throw. The LVA-based rules (not SE) would otherwise receive some incomplete results and would continue their computation. 
```suggestion
CancellationToken.ThrowIfCancellationRequested();
```